deploy/kubernetes/base/rbac/rbac.yaml

```diff
@@ -73,7 +46,7 @@ metadata:
 subjects:
   - kind: ServiceAccount
     name: csi-controller-sa
-    namespace: default
+    namespace: pdcsi-serving
```

Comment:
same comment about `pdcsi`, I would recommend something more like `pd-driver-` and what does `serving` mean, that's not a term used anywhere else that I've seen